examples/plot/app.js

```diff
@@ -15,8 +15,9 @@ class Root extends Component {
       viewport: {
         lookAt: [0, 0, 0],
         distance: 3,
-        rotationX: -30,
-        rotationY: 30,
+        pitchAngle: -30,
```

Comment:
I prefer we follow only one convention either X, Y and Z or Pitch, Roll and Yaw.  `pitchAngle` and `orbitAxis: 'Y'` seem to combine both.

Reply:
actually there is only one convention, Pitch is the rotating angle around X, here I use pitchAngle to name the rotation angle around X, but use Y to specify the axis for orbit. So there is no conflict in this area.

Reply:
Is orbit axis Y the perpendicular axis? so what if the axis for orbit is not Y but X or Z. Will the pitch lead to a different rotation direction?

Reply:
By default, 'Z' is the perpendicular axis, we can set 'Y' as the orbit axis, too. But can't set 'X' as the orbit axis cause it is not meaningful in right handed 3D coordinate system. For both 'Z' and 'Y' cases, the pitch is consistent. Will add more comments in the code.